lib/chorde/sPickle.py

```diff
@@ -27,6 +27,7 @@
 import struct
 import threading
 from binascii import hexlify, unhexlify
+from past.builtins import unicode
```

Comment:
There's no need to be dual-python compatible, python 2.x support has been dropped